src/index.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", function() {
+
+let currentTemp = 72;
+let sky = "";
+// let defaultCityName = "What city are you searching for?";
+let defaultCityName = "";
+
+
+function getTemp() {
+    const temperature = document.getElementById("temp");
+    temperature.textContent = currentTemp;
+    if (currentTemp >= 80) {
+        temperature.className = "hot";
+    } else if (currentTemp >= 70) {
+        temperature.className = "warm";
+    } else if (currentTemp >= 60) {
+        temperature.className = "medium";
+    } else if (currentTemp >= 50) {
+        temperature.className = "cool";
+    } else {
+        temperature.className = "cold";
+    }
+}
+
+const button1 = document.getElementById("increase_temperature");
+button1.addEventListener("click", () => {
+    currentTemp = currentTemp + 1;
+    getTemp();
+    getLandscape();
+});
+
+const button2 = document.getElementById("decrease_temperature");
+button2.addEventListener("click", () => {
+    currentTemp = currentTemp - 1;
+    getTemp();
+    getLandscape();
+});
+
+const getLandscape = () => {
```

Comment:
The if conditions here are exactly the same as `getTemp()`. Instead of having a separate function, the logic for updating the landscape can be done in the `getTemp()` function to make the code DRY-er.